python/README.md

```diff
@@ -26,4 +26,5 @@ optional arguments:
   -query SQLQUERY, --sqlquery SQLQUERY SQL query to test
   -tls, --tls Enable encrypted connection
   -certs TRUSTEDCERTIFICATES, --trustedCertificates TRUSTEDCERTIFICATES Path to trusted certificates for encrypted connection
+  -sn SERVERNAME, --servername SERVERNAME to override hostname with
```

Comment:
Can the naming be changed indicate the flag will change the hostname for SNI only?